Connnect/JoinViewController.swift

```diff
@@ -16,10 +16,7 @@ class JoinViewController: UIViewController {
     override func viewDidLoad() {
```

Comment:
Now that there is no code here anymore we could lose the `viewDidLoad` function too.